client/metric_dashboards.go

```diff
@@ -7,18 +7,18 @@ import (
 	"net/http"
 )
 
-type MetricDashboard struct {
-	Type       string                    `json:"type"`
-	ID         string                    `json:"id"`
-	Attributes MetricDashboardAttributes `json:"attributes,omitempty"`
+type UnifiedDashboard struct {
```

Comment:
This file has only renaming changes to avoid the confusing scenario where we would otherwise be calling a function or using a struct with "metric" in the name even though it applies to any dashboard.